jet_fuel/config/application.rb

```diff
@@ -0,0 +1,26 @@
+require File.expand_path('../boot', __FILE__)
+
+require 'rails/all'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.